client/pages/sign-up.jsx

```diff
@@ -61,83 +63,111 @@ export default class SignUp extends React.Component {
       err = '';
     }
     return (
-     <div className="list-background">
+      <div className="list-background">
         <div>
           <h1>Sign Up</h1>
         </div>
-      <div className="full-column">
-        <div className="row center">
-          <form onSubmit={this.handleSubmit}>
-            <div>
-              <label>
-                <h2>
-                  Username
-                </h2>
-              </label>
-              <input
-                required
-                id="username"
-                type="text"
-                name="username"
-                onChange={this.handleChange}
-                className="username"
-              />
+        <div className="full-column">
+          <div className="width-60 auto">
+            <div className="row center">
+              <form onSubmit={this.handleSubmit}>
+                <div className="row space-between margin-vert-2rem">
+                  <div className="margin-vert-2rem">
+                    <label>
+                      <h2>
+                        Username
+                      </h2>
+                    </label>
+                    <input
+                      required
+                      id="username"
+                      type="text"
+                      name="username"
+                      onChange={this.handleChange}
+                      className="username"
+                    />
+                    <label>
+                      <h2>
+                        Password
+                      </h2>
+                    </label>
+                    <input
+                      required
+                      id="password"
+                      type="password"
+                      name="password"
+                      onChange={this.handleChange}
+                      className="password"
+                    />
+                    <p className="sign-in-no-account">
+                      Already have an account?
+                    </p>
+                    <a href="#sign-in" className="sign-in-no-account">
+                      Sign In
+                    </a>
+                  </div>
+                  <div className="margin-vert-2rem">
+                    <div>
+                      <label>
+                        <h2>
+                          Phone
+                        </h2>
+                      </label>
+                      <input
+                        required
+                        id="phone"
+                        type="number"
```

Comment:
There is a `type="tel"` that you can use for telephone inputs.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel